codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonHelper.java

```diff
@@ -105,7 +105,7 @@ private void loadClassToCache(String key, Class clazz, String methodName) {
         Method jsonMethod = null;
         Method[] methods = clazz.getMethods();
         for (Method method : methods) {
-            if (methodName.equals(method.getName())) {
+            if (methodName.equals(method.getName()) && !method.isBridge()) {
```

Comment:
Is there more than one `bridgeMethod`? What should be done in such cases?

Reply:
here is just for the subclass of a generic interface, because there are two methods in the subclass, one is normal, the other is a bridge method without real type, so we must filter this to get the real method we want.